### PR TITLE
Toggle on migrations details by setting default filtering value to false

### PR DESCRIPTION
**Pull request by HaudinFlorence**
PR Checklist:

- [ ] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [ ] if you are adding a new page under `docs/` or `community/`, you have added it to the sidebar in the corresponding `_sidebar.json` file
- [x] put any other relevant information below

This PR aims at fixing task 6 of issue https://github.com/conda-forge/conda-forge.github.io/issues/2137

### CODE REVIEW

**Comment by beckermr** [comment]
@jaimergp @HaudinFlorence Maybe we set the default for the "done" category to not display them?

**Comment by HaudinFlorence** [comment]
> @jaimergp @HaudinFlorence Maybe we set the default for the "done" category to not display them?

I added it in the last commit. But I am not sure this is the nicer way to do it.

File: src/pages/status/migration/index.jsx [hunk 1]
```diff
@@ -200,8 +203,10 @@ function Graph(props) {
 }
 
 function Table({ details }) {
-  const [filters, setState] = useState(ORDERED
-      .reduce((filters, [status]) => ({ ...filters, [status]: true }), {}));
+  const defaultFilters = ORDERED.reduce((filters, [status]) => ({ ...filters, [status]: DEFAULT_FILTERING_STATE }), {});
+  const [filters, setState] = useState(defaultFilters);
+  filters['done'] = !DEFAULT_FILTERING_STATE;
```
Comment (jaimergp) [inline src/pages/status/migration/index.jsx:208]: This way makes "Done" always off, I think. I can't toggle it anymore, at least. Let me add a suggestion.

File: src/pages/status/migration/index.jsx [hunk 2]
```diff
@@ -16,6 +16,9 @@ const ORDERED = [
   ["bot-error", "Bot error"],
 ];
```
Comment (jaimergp) [inline src/pages/status/migration/index.jsx:17]: What if we add a third field here, with their default state (True=shown, False=hidden).

File: src/pages/status/migration/index.jsx [hunk 3]
```diff
@@ -200,8 +203,10 @@ function Graph(props) {
 }
 
 function Table({ details }) {
-  const [filters, setState] = useState(ORDERED
-      .reduce((filters, [status]) => ({ ...filters, [status]: true }), {}));
+  const defaultFilters = ORDERED.reduce((filters, [status]) => ({ ...filters, [status]: DEFAULT_FILTERING_STATE }), {});
```
Comment (jaimergp) [inline src/pages/status/migration/index.jsx:206]: And then the defaultFilters is just collecting that third field here.

File: src/pages/status/migration/index.jsx (hunk 2 above)
Comment (HaudinFlorence) [reply src/pages/status/migration/index.jsx:17]: I thought about adding a third field here too.

File: src/pages/status/migration/index.jsx (hunk 1 above)
Comment (HaudinFlorence) [reply src/pages/status/migration/index.jsx]: Yes. That's not what we want.

File: src/pages/status/migration/index.jsx [hunk 4]
```diff
@@ -8,13 +8,13 @@ import styles from "./styles.module.css";
...
-  ["not-solvable", "Not solvable"],
-  ["bot-error", "Bot error"],
-];
+  ["done", "Done", true],
+  ["in-pr", "In PR", false],
+  ["awaiting-pr", "Awaiting PR", false],
+  ["awaiting-parents", "Awaiting parents", false],
+  ["not-solvable", "Not solvable", false],
+  ["bot-error", "Bot error", false],
+]
```
Comment (jaimergp) [inline src/pages/status/migration/index.jsx:17]:
```suggestion
];
```

File: src/pages/status/migration/index.jsx [hunk 5]
```diff
@@ -8,13 +8,13 @@ import styles from "./styles.module.css";
 
 // { Done, In PR, Awaiting PR, Awaiting parents, Not solvable, Bot error }
 const ORDERED = [
```
Comment (jaimergp) [inline src/pages/status/migration/index.jsx:10]: Can we add a comment about the boolean meaning here?
```suggestion
// The third value is a boolean representing the default display state on load
// 'true' means visible, 'false' means hidden
const ORDERED = [
```

File: src/pages/status/migration/index.jsx [hunk 6]
```diff
@@ -7,13 +7,15 @@ import SVG from 'react-inlinesvg';
 import styles from "./styles.module.css";
 
 // { Done, In PR, Awaiting PR, Awaiting parents, Not solvable, Bot error }
+// The third value is a boolean representing the default display state on load
+// 'true' means visible, 'false' means hidden
```
Comment (jaimergp) [inline src/pages/status/migration/index.jsx:11]:
```suggestion
// 'true' means hidden, 'false' means visible
```

**Review by jaimergp — APPROVED** [review]

File: src/pages/status/migration/index.jsx [hunk 7]
```diff
@@ -200,8 +202,8 @@ function Graph(props) {
 }
 
 function Table({ details }) {
-  const [filters, setState] = useState(ORDERED
-      .reduce((filters, [status]) => ({ ...filters, [status]: true }), {}));
+  const defaultFilters = ORDERED.reduce((filters, [status], index) => ({ ...filters, [status]: ORDERED[index][2] }), {});
```
Comment (afshin) [inline src/pages/status/migration/index.jsx:205]:
```suggestion
  const defaultFilters = ORDERED.reduce((filters, [status, _, toggled], index) => ({ ...filters, [status]: toggled }), {});
```

**Comment by jaimergp** [comment]
Thanks!

**Pull request merged**